extensions/hacker-news-500/src/view-500.tsx

```diff
@@ -0,0 +1,114 @@
+import { useEffect, useState } from "react";
+import { Cache, Icon, MenuBarExtra, open, getPreferenceValues } from "@raycast/api";
+import { getStories } from "./hackernews";
+import { Story } from "./types";
+
+const cache = new Cache();
+// Stories that the user clicked on
+const readKey = "read-stories";
+// Stories that we've seen over the past week.
+// If we've seen something more than 24 hours ago, we remove it
+// This allows for stories posted e.g. 3 days ago to hit 500 points today
+// [{ story: {}, seen: 1234567890 }]
+const seenKey = "seen-stories";
+// To cache the points to clear cache when they change
+const prefKey = "preferences";
```

Comment:
Is this key a little too generic? How come we cache the preferences?

Reply:
Isnt the cache scoped to the extension? The point of caching them is that if the points preference changes, I need to clear the other cache

Reply:
Is there a better way to know when the user changes a preference?